binary_transparency/firmware/internal/client/client.go

```diff
@@ -32,16 +32,21 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-// Client is an HTTP client for the FT personality.
+// ReadonlyClient is an HTTP client for the FT personality.
 //
 // TODO(al): split this into Client and SubmitClient.
-type Client struct {
+type ReadonlyClient struct {
```

Comment:
Only minor comment is whether ReadonlyClient or ReadOnlyClient. But whatever :-)

Reply:
I think `readonly` is one word, so I'm gonna run with this for now :)